statsd/client.py

```diff
@@ -197,6 +197,23 @@ def reconnect(self, data):
         self.connect()
 
 
+class UnixSocketStatsClient(TCPStatsClient):
+    """Unix domain socket version of TCPStatsClient."""
+
+    def __init__(self, socket_path='/var/run/statsd.socket', prefix=None,
```

Comment:
i don't think there's any appropriate default here, since there's not a default in the server implementation